File: carbon/src/main/java/carbon/view/RevealView.java
package carbon.view;

import android.animation.Animator;

public interface RevealView extends RenderingModeView {
    float MAX_RADIUS = -1;

    Animator createCircularReveal(android.view.View hotspot, float startRadius, float finishRadius);

    Animator createCircularReveal(int x, int y, float startRadius, float finishRadius);
}

File: carbon/src/main/java/carbon/view/RevealView.kt
package carbon.view

import android.animation.Animator

interface RevealView {

    fun createCircularReveal(hotspot: android.view.View, startRadius: Float, finishRadius: Float): Animator

    fun createCircularReveal(x: Int, y: Int, startRadius: Float, finishRadius: Float): Animator

    companion object {
        const val MAX_RADIUS = -1f
    }
}
